contributing.rst

```diff
@@ -37,6 +37,10 @@ General commit rules
    characters, separated by single blank line from the other parts (including
    the subject).
 
+4. Don't squash commits in your pull request. It makes it harder to
```

Comment:
Debatable. I only look at "files changed" and not at "commits" because I cannot apply diff on top of diff on top of diff and imagine what the result looks like.